source/elements/oneMKL/source/domains/spblas/data_types/set_coo_matrix_data.rst

```diff
@@ -52,12 +52,12 @@ set_coo_matrix_data (Buffer version)
    .. rubric:: Template parameters
 
    dataType
-      See :ref:`supported template types<onemkl_sparse_supported_types>`. Can be
-      a different type than what was used when creating the ``matrix_handle_t``.
+      See :ref:`supported template types<onemkl_sparse_supported_types>`. Must
+      be the same type than what was used when creating the ``matrix_handle_t``.
```

Comment:
```suggestion
      See :ref:`supported template types<onemkl_sparse_supported_types>`. Must
      be the same type as was used when creating the ``matrix_handle_t``.
```

And similar elsewhere.

Reply:
Thank you, I missed many references that I meant to change as well!
This is fixed in https://github.com/uxlfoundation/oneAPI-spec/pull/542/commits/1fdd42d6cb0c400b1ff7c4821a63d6bff1c79d4b